### PR TITLE
Add Trino forum post

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2023-08-21-starburst-forum-trino-questions-answers.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+[announced the launch of a community forum](https://www.starburst.io/blog/starburst-community-forum/)
+with a [Trino category](https://www.starburst.io/community/forum/c/trino/7).
+Here in the Trino space, we're excited about the announcement, as it means that
+there will be a dedicated, easy-to-search site for knowledge-sharing and having
+your Trino questions answered. The Trino [Slack]({{site.url}}/slack.html) (also
+sponsored by Starburst) has been the key solution for community discussion and
+questions for a while, but with the friction of needing to sign up and Slack's
+notoriously poor search functionality limiting discoverability of past
+conversations, it's never been the ideal way to spread information. With a
+[Discourse-powered](https://www.discourse.org/)forum tailored to Trino
```
Comment (mosabua) [inline _posts/2023-08-21-starburst-forum-trino-questions-answers.md:20]:
```suggestion
[Discourse-powered](https://www.discourse.org/) forum tailored to Trino
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
I think we still need to point out that the main Trino community places are slack and github. And the forum is a Starburst thing. We also should get the sponsor PR merged before this.

File: _posts/2023-08-21-starburst-forum-trino-questions-answers.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+show_pagenav: false
+---
+
+Last week, one of Trino's main sponsors,[Starburst Data](https://www.starburst.io/),
+[announced the launch of a community forum](https://www.starburst.io/blog/starburst-community-forum/)
+with a [Trino category](https://www.starburst.io/community/forum/c/trino/7).
+Here in the Trino space, we're excited about the announcement, as it means that
+there will be a dedicated, easy-to-search site for knowledge-sharing and having
+your Trino questions answered. The Trino [Slack]({{site.url}}/slack.html) (also
+sponsored by Starburst) has been the key solution for community discussion and
```
[the remark this answers is not in the document]
Comment (colebow) [reply _posts/2023-08-21-starburst-forum-trino-questions-answers.md:16]: Long sentences are ok from time to time, and this one is all one cohesive thought, so I think I'm gonna keep as-is.

[the remark this answers is not in the document]
Comment (mosabua) [reply _posts/2023-08-21-starburst-forum-trino-questions-answers.md:16]: I don't agree necessarily, but I also good with leaving it.

**Pull request closed without merging**